File: 0815/C#/fizzbuzz.cs
using System;

public class Fizzbuzz
{
    public static void Main(string[] args)
    {
        for(int i = 1;i<=100;i++){
            
            if(i % 3 == 0.00 && i % 5 == 0.00) {
                Console.WriteLine("Fizzbuzz");
            }
            else if (i % 3 == 0.00){
                Console.WriteLine("Fizz");
            }
            else if (i % 5 == 0.00){
                Console.WriteLine("buzz");
            }
            else {
                Console.WriteLine(i);
            }
        }
    }
}

File: 0815/Java/Fizzbuzz.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        try {
            
            int input = Integer.parseInt(scanner.nextLine()); // Convert input to int
            
            if(input % 3 == 0 && input % 5 == 0){
                System.out.println("Fizzbuzz"); // Output the input
            }
            else if(input % 3 == 0){
                System.out.println("Fizz"); // Output the input
            }
            else if(input % 5 == 0){
                System.out.println("buzz"); // Output the input
            }
            else{
                System.out.println(input); // Output the input
            }
        } catch (NumberFormatException e) {
            System.out.println("Invalid input. Please enter a valid number."); // Handle invalid input
        } finally {
            scanner.close(); // Close the scanner (optional)
        }
    }
}